crates/ruff_workspace/src/options.rs

```diff
@@ -2238,6 +2266,20 @@ impl Pep8NamingOptions {
 #[serde(deny_unknown_fields, rename_all = "kebab-case")]
 #[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
 pub struct PycodestyleOptions {
+    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`doc-line-too-long`] violations within
```

Comment:
nit: for consistency
```suggestion
    /// The maximum [display width](http://www.unicode.org/reports/tr11/#Overview) to allow for [`doc-line-too-long`] violations within
```